### PR TITLE
FullSecuredViewController inheritance

### PR DESCRIPTION
**Pull request by synchuk2010**
### Jira references
- Ticket (issue) — https://essentiatechnologies.atlassian.net/browse/MB-383
### Description
- Move FullSecuredViewController inheritance to BaseTableAdapterViewController

### CODE REVIEW

**Comment by ghost** [comment]
## Black Duck Security Report

Merging #100 into develop will not change security risk.

### Added Components

Clean:       5

[Click here to see full report](https://copilot.blackducksoftware.com/github/repos/essentiaone/Essentia-iOS/pull-requests/100)

File: Essentia/UserCases/Settings/Controllers/FullSecuredViewController.swift [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+            .actionCenteredButton(title: LS("Settings.FullSecured.Continue"),
+                                  action: continueAction,
+                                  textColor: .black,
+                                  backgrount: .white),
+            .empty(height: 24, background: .clear)
+        ]
+    }
+    
+    private func applyDesign() {
+        tableView.backgroundColor = RGB(59, 207, 85)
```
Comment (impul) [inline Essentia/UserCases/Settings/Controllers/FullSecuredViewController.swift]: Please use AppColorInterface instead

File: Essentia/UserCases/Settings/Controllers/FullSecuredViewController.swift [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        buttonsState
+    }
+    
+    private var buttonsState: [TableComponent] {
+        if !isAnimationShow { return [] }
+        return [
+            .titleWithFontAligment(font: AppFont.bold.withSize(42),
+                                   title: LS("Settings.FullSecured.Title"),
+                                   aligment: .center,
+                                   color: .white),
```
Comment (impul) [inline Essentia/UserCases/Settings/Controllers/FullSecuredViewController.swift]: The same, all colours except `.clear` should be used via App Color Interface

File: Essentia/UserCases/Settings/Controllers/FullSecuredViewController.swift [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    private func applyDesign() {
+        tableView.backgroundColor = RGB(59, 207, 85)
+        tableView.bounces = false
+        tableView.isScrollEnabled = false
+    }
+    
+    
+    private func showBankAnimation() {
+        let animation = PNGAnimation.securing99toSafe
+        DispatchQueue.main.asyncAfter(deadline: DispatchTime.now() + animation.animationDuration - 0.63) { [unowned self] in
```
Comment (impul) [inline Essentia/UserCases/Settings/Controllers/FullSecuredViewController.swift]: Try to encapsulate this logic, make it reusable

**Review by impul — CHANGES_REQUESTED** [review]

File: Essentia/UserCases/Settings/Controllers/FullSecuredViewController.swift (hunk 1 above)
Comment (synchuk2010) [reply Essentia/UserCases/Settings/Controllers/FullSecuredViewController.swift]: done

File: Essentia/UserCases/Settings/Controllers/FullSecuredViewController.swift (hunk 2 above)
Comment (synchuk2010) [reply Essentia/UserCases/Settings/Controllers/FullSecuredViewController.swift]: done

File: Essentia/UserCases/Settings/Controllers/FullSecuredViewController.swift (hunk 3 above)
Comment (synchuk2010) [reply Essentia/UserCases/Settings/Controllers/FullSecuredViewController.swift]: see, 4e24370

**Review by impul — APPROVED** [review]

**Pull request merged**